python/sparktk/models/timeseries/arimax.py

```diff
@@ -0,0 +1,305 @@
+
+        >>> predicted_frame.inspect(columns=["CO_GT","predicted_y"])
+        [#]  CO_GT  predicted_y
+        =========================
+        [0]    2.6  None
+        [1]    2.0  2.38533563132
+        [2]    2.2   2.2572598393
+        [3]    2.2  2.22538778928
+        [4]    1.6  2.21745633278
+        [5]    1.2  2.21548256573
+        [6]    1.2  2.21499138781
+        [7]    1.0  2.21486915669
+        [8]    2.9   2.2148387391
+        [9]    2.6  2.21483116959
+
+    """
+    def __init__(self, tc, scala_model):
+        self._tc = tc
+        tc.jutils.validate_is_jvm_instance_of(scala_model, _get_scala_obj(tc))
+        self._scala = scala_model
+
+    @staticmethod
+    def _from_scala(tc, scala_model):
+        """
+        Load an ARIMAX model
+
+        :param tc: (TkContext) Active TkContext
+        :param scala_model: (scala ARIMAXModel) Scala model to load.
+        :return: (ARIMAXModel) ARIMAXModel object
+        """
+        return ArimaxModel(tc, scala_model)
+
+    @property
+    def p(self):
+        """
+        Autoregressive order
+        """
+        return self._scala.p()
+
+    @property
+    def d(self):
+        """
+        Differencing order
+        """
+        return self._scala.d()
+
+    @property
+    def q(self):
+        """
+        Moving average order
+        """
+        return self._scala.q()
+
+    @property
+    def x_max_lag(self):
+        """
+        The maximum lag order for exogenous variables.
+        """
+        return self._scala.xregMaxLag()
+
+    @property
+    def includeIntercept(self):
+        """
+        A boolean flag indicating if the intercept should be included.
+        """
+        return self._scala.includeIntercept()
+
+    @property
+    def includeOriginalXreg(self):
+        """
+        A boolean flag indicating if the non-lagged exogenous variables should be included.
+        """
+        return self._scala.includeOriginalXreg()
+
+    @property
+    def init_params(self):
+        """
+        A set of user provided initial parameters for optimization
+        """
+        return self._scala.initParams()
+
+    @property
+    def c(self):
+        """
+        Incercept
+        """
+        return self._scala.c()
+
+    @property
+    def ar(self):
+        """
+        Coefficient values from the trained model (AR with increasing degrees).
+        """
+        return list(self._tc.jutils.convert.from_scala_seq(self._scala.ar()))
+
+    @property
+    def ma(self):
+        """
+        Coefficient values from the trained model (MA with increasing degrees).
+        """
+        return list(self._tc.jutils.convert.from_scala_seq(self._scala.ma()))
+
+    @property
+    def xreg(self):
+        """
+        Coefficient values from the trained model fox exogenous variables with increasing degrees.
+        """
+        return list(self._tc.jutils.convert.from_scala_seq(self._scala.xreg()))
+
+
+    def predict(self, frame, ts_column, x_columns):
```

Comment:
per sparktk convention, predict should add a column with predictions and return nothing, otherwise name this create_predict_frame